services/src/unit-test/java/uk/gov/moj/sdt/services/RetryMessageAlreadySentServiceTest.java

```diff
@@ -59,170 +60,172 @@
-        EasyMock.expect(this.mockCacheable.getValue(IGlobalParameter.class, "MAX_FORWARDING_ATTEMPTS")).andReturn(
+        when(this.mockCacheable.getValue(IGlobalParameter.class, MAX_FORWARDING_ATTEMPTS)).thenReturn(
                 maxForwardingAttemptsParam);
 
-        final int maxForwardingAttempts = Integer.valueOf(maxForwardingAttemptsParam.getValue());
+        final int maxForwardingAttempts = Integer.parseInt(maxForwardingAttemptsParam.getValue());
 
-        EasyMock.expect(mockIndividualRequestDao.getPendingIndividualRequests(maxForwardingAttempts)).andReturn(
+        when(mockIndividualRequestDao.getPendingIndividualRequests(maxForwardingAttempts)).thenReturn(
                 individualRequests);
 
         for (IIndividualRequest individualRequestObj : individualRequests) {
             mockMessagingUtility.enqueueRequest(individualRequestObj);
-            EasyMock.expectLastCall();
 
             // Re-set the forwarding attempts on the individual request.
             individualRequestObj.resetForwardingAttempts();
         }
 
         mockIndividualRequestDao.persistBulk(individualRequests);
-        EasyMock.expectLastCall();
-
-        EasyMock.replay(mockIndividualRequestDao);
-        EasyMock.replay(mockCacheable);
-        EasyMock.replay(mockMessagingUtility);
 
         this.messageTaskService.queueMessages();
 
-        EasyMock.verify(mockIndividualRequestDao);
-        EasyMock.verify(mockCacheable);
-        EasyMock.verify(mockMessagingUtility);
+        verify(mockIndividualRequestDao).getPendingIndividualRequests(maxForwardingAttempts);
+        verify(mockCacheable).getValue(IGlobalParameter.class, MAX_FORWARDING_ATTEMPTS);
+        verify(mockMessagingUtility, times(2)).enqueueRequest(any());
 
-        Assert.assertEquals("Forwarding attempts on individual request", 0, individualRequests.get(0)
-                .getForwardingAttempts());
-        Assert.assertEquals("Status set to Receieved", IIndividualRequest.IndividualRequestStatus.RECEIVED
-                .getStatus(), individualRequests.get(0).getRequestStatus());
-
-        Assert.assertTrue("Test completed succesfully", true);
+        assertEquals(0, individualRequests.get(0)
+                .getForwardingAttempts(), FORWARDING_ATTEMPTS_ON_INDIVIDUAL_REQUEST);
+        assertEquals(IIndividualRequest.IndividualRequestStatus.RECEIVED
+                .getStatus(), individualRequests.get(0).getRequestStatus(), STATUS_SET_TO_RECEIVED);
 
+        assertTrue(true, TEST_COMPLETED_SUCCESSFULLY);
     }
 
     /**
      * This method tests the all successful scenario of the queue pending message
      * when the forwarding attempts of individual request has exceeded the max forwarding attempts.
      */
     @Test
-    public void queuePendingMessageExceededForwardingAttempts() {
-        final String sdtRequestRef = "TEST_1";
+    void queuePendingMessageExceededForwardingAttempts() {
+        final String sdtRequestRef = TEST_1;

```

Comment:
As a constant has been created for the SDT request value this variable is redundant. It can be removed and the constant used in it's place.

Reply:
done

Reply:
Confirmed constant now used in place of variable.